include/openssl/kdf.h

```diff
@@ -25,12 +25,32 @@ extern "C" {
 // and writes them to |out|. It returns one on success and zero on error.
 // TLS 1.2: https://datatracker.ietf.org/doc/html/rfc5246#section-5
 // TLS 1.{0,1}: https://datatracker.ietf.org/doc/html/rfc4346#section-5
-OPENSSL_EXPORT int CRYPTO_tls1_prf(const EVP_MD *digest,
-                                   uint8_t *out, size_t out_len,
-                                   const uint8_t *secret, size_t secret_len,
-                                   const char *label, size_t label_len,
-                                   const uint8_t *seed1, size_t seed1_len,
-                                   const uint8_t *seed2, size_t seed2_len);
+OPENSSL_EXPORT int CRYPTO_tls1_prf(const EVP_MD *digest, uint8_t *out,
+                                   size_t out_len, const uint8_t *secret,
+                                   size_t secret_len, const char *label,
+                                   size_t label_len, const uint8_t *seed1,
+                                   size_t seed1_len, const uint8_t *seed2,
+                                   size_t seed2_len);
+
+// SSKDF_digest computes the One-step (Single-step) key derivation using the
+// provided digest algorithm as the backing PRF.
+//
+// Returns a 1 on success, otherwise returns 0.
+OPENSSL_EXPORT int SSKDF_digest(uint8_t *out_key, size_t out_len,
```

Comment:
It would be good to describe the arguments in relation to the standard.
I believe the standard allows _L_ to be an arbitrary length in bits, not necessarily divisible by 8. But I think we don't neither does OpenSSL. I recommend we state that.
Same for `SSKDF_hmac`.